test/files/neg/warn-unused-params.scala

```diff
@@ -67,3 +67,7 @@ class Reusing(u: Int) extends Unusing(u)   // no warn
 class Main {
   def main(args: Array[String]): Unit = println("hello, args")  // no warn
 }
+
+trait Unimplementation {
+  def f(u: Int): Int = ???        // no warn for param in unimplementation
+}
```

Comment:
Pedantically, wouldn't an "unimplementation" be when you override a concrete method like this?

Reply:
"Deimplementation." But I see it's `NotImplementedError` and not `UnimplementedException`. Unfortunately, it is final.

Reply:
Also, the more modest April 1 post would have announced that Seth had persuaded Martin to rename `NotImplementedError` to `UnimplementedMistake`, and how this represented a huge advance in the face of Martin's conservative approach to language design and backward compatibility. It took a long session at the Biergarten and all of Seth's charms. Basically he drank him under the table and agreed not to post anything on twitter.

Reply:
> It took a long session at the Biergarten and all of Seth's charms. Basically he drank him under the table and agreed not to post anything on twitter.

And that's _not_ how we got Setha?